var/spack/repos/builtin/packages/py-pyqt4/package.py

```diff
@@ -25,19 +26,77 @@ class PyPyqt4(SIPPackage):
     ]
 
     version('4.12.3', sha256='a00f5abef240a7b5852b7924fa5fdf5174569525dc076cd368a566619e56d472')
-    version('4.11.3', '997c3e443165a89a559e0d96b061bf70',
+    version('4.11.3', sha256='853780dcdbe2e6ba785d703d059b096e1fc49369d3e8d41a060be874b8745686',
             url='http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt-x11-gpl-4.11.3.tar.gz')
 
+    variant('qsci', default=False, description='Build with QScintilla python bindings')
+
     # Supposedly can also be built with Qt 5 compatibility layer
     depends_on('qt@:4')
+    depends_on('qscintilla', when='+qsci')
+
+    # For building Qscintilla python bindings
+    resource(name='qscintilla',
+             url='https://www.riverbankcomputing.com/static/Downloads/QScintilla/2.10.2/QScintilla_gpl-2.10.2.tar.gz',
+             sha256='14b31d20717eed95ea9bea4cd16e5e1b72cee7ebac647cba878e0f6db6a65ed0',
+             destination='spack-resource-qscintilla',
+             when='^qscintilla@2.10.2'
+    )
 
     # https://www.riverbankcomputing.com/static/Docs/PyQt4/installation.html
     def configure_file(self):
         return 'configure-ng.py'
 
     def configure_args(self):
-        return [
+        args = [
             '--pyuic4-interpreter', self.spec['python'].command.path,
             '--sipdir', self.prefix.share.sip.PyQt4,
-            '--stubsdir', join_path(site_packages_dir, 'PyQt4'),
+            '--stubsdir', join_path(site_packages_dir, 'PyQt4')
         ]
+        if '+qsci' in self.spec:
+            args.extend(['--qsci-api-destdir', self.prefix.share + '/qsci'])
+        return args
+
+    @run_after('install')
+    def make_qsci(self):
+        if '+qsci' in self.spec:
+            rsrc_py_path = os.path.join(
+                str(self.stage.source_path),
+                'spack-resource-qscintilla/QScintilla_gpl-',
+                str(self.spec['qscintilla'].version), 'Python')
+            with working_dir(rsrc_py_path):
+                pydir = join_path(site_packages_dir, 'PyQt4')
+                python = self.spec['python'].command
+                python('configure.py',
+                       '--sip=' + self.prefix.bin.sip,
+                       '--qsci-incdir=' +
+                       self.spec['qscintilla'].prefix.include,
+                       '--qsci-libdir=' + self.spec['qscintilla'].prefix.lib,
+                       '--qsci-sipdir=' + self.prefix.share.sip.PyQt4,
+                       '--apidir=' + self.prefix.share.qsci,
+                       '--destdir=' + pydir,
+                       '--pyqt-sipdir=' + self.prefix.share.sip.PyQt4,
+                       '--sip-incdir=' + self.prefix + '/include/python' +
+                       str(self.spec['python'].version.up_to(2)),
```

Comment:
I think you can just use `python_include_dir` here, it's added to the module via the Python package.